### PR TITLE
Add coredns to kvm

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/4452

Enable coredns management via cluster-operator for kvm.

### CODE REVIEW

File: service/controller/kvm/v8/key/key.go [hunk 1]
```diff
@@ -3,14 +3,25 @@ package key
...
+		{
+			AppName:           "coredns",
+			ChannelName:       "0-2-stable",
+			ChartName:         "kubernetes-coredns-chart",
+			ConfigMapName:     "coredns-values",
+			Namespace:         metav1.NamespaceSystem,
+			ReleaseName:       "coredns",
+			UserConfigMapName: "coredns-user-values",
+		},
+	}
```
Comment (kopiczko) [inline service/controller/kvm/v8/key/key.go]: Don't we need to disable coredns in cc also?

Comment (TheoBrigitte) [reply service/controller/kvm/v8/key/key.go]: Yes we need to disable coredns from cloudconfig for kvm, which is handled here https://github.com/giantswarm/kvm-operator/pull/604

File: service/controller/kvm/v8/version_bundle.go [hunk 2]
```diff
@@ -7,13 +7,22 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "coredns",
+				Description: "Add coredns chartconfig.",
```
Comment (TheoBrigitte) [inline service/controller/kvm/v8/version_bundle.go]: This message is not very explicit on what adding coredns means, how about:
Add management of CoreDNS component for tenant clusters. Also enable CoreDNS custom configuration.

Comment (TheoBrigitte) [reply service/controller/kvm/v8/version_bundle.go]: Or align with aws and set same message: `Allow advanced CoreDNS configuration. See https://docs.giantswarm.io/guides/advanced-coredns-configuration`

Comment (teemow) [reply service/controller/kvm/v8/version_bundle.go]: Yes please align with aws.

Comment (TheoBrigitte) [reply service/controller/kvm/v8/version_bundle.go]: done

**Pull request merged**